src/pages/[platform]/build-a-backend/storage/download/index.mdx

```diff
@@ -85,10 +85,10 @@ console.log('URL expires at: ', getUrlResult.expiresAt);
 import { getUrl } from 'aws-amplify/storage';
 
 const getUrlResult = await getUrl({
-  key: 'abc.txt',
+  key: 'album/2024/1.jpg',
   options: {
     accessLevel?: 'guest' , // can be 'private', 'protected', or 'guest' but defaults to `guest`
-    targetIdentityId?: 'XXXXXXX', // id of another user, if `accessLevel` is `guest`
+    targetIdentityId?: 'XXXXXXX', // id of another user, if `accessLevel` is `protected`
```

Comment:
Nit: Should we change `accessLevel` to `protected` to make it make more sense as a complete example?